### ISSUE TITLE
The game crashes on mac since the new update

### ISSUE THREAD

**Issue by 3Yordano3** [ValveSoftware/Dota2-Gameplay#8083]
### Description

After 5 seconds the game crashes without any error message

**Comment by CassMall23** [comment]
I have this problem too.

**Comment by CassMall23** [comment]
I deleted the "heroes" folder in the "panorama" section and now it lets me into the main menu.

**Comment by TooSpicyToWork** [comment]
> I deleted the "videos" folder in the "panorama" section and now it lets me into the main menu. I immediately register map_enable_background_maps 0 in the console and now I can sit in the main menu as much as I want. But the game crashes when you try to start the game in the created lobby.

Yeah same with me, it's really weird. Before the update, I could play and start games normally but after the update, I cant even start demo mode without a crash. It just crashes on the loading screen before the game even starts.

**Comment by TooSpicyToWork** [comment]
I updated my Mac to the latest version via software update to Monterey 12.6.5 (was Mojave 10.14 before the update), and I started the game with just the launch option -GL and it didn't crash on the menu screen. On top of that, I started a bot game and it didn't crash! So maybe the solution is to update your Mac to the latest version.

**Comment by CassMall23** [comment]
@TooSpicyToWork Can you write all the startup parameters that you had when starting dota?

**Comment by TooSpicyToWork** [comment]
> @TooSpicyToWork Can you write all the startup parameters that you had when starting dota?

Yeah sure. They were:

-novid -high -map dota -noprewarm +map_enable_background_maps 0

![Screen Shot 2023-04-21 at 2 34 10 pm](https://user-images.githubusercontent.com/131443012/233767603-8e84b6ed-281f-4fa8-b8bb-0314e870ff1c.png)

**Comment by ilkedemir** [comment]
> > @TooSpicyToWork Can you write all the startup parameters that you had when starting dota?
> 
> Yeah sure. They were:
> 
> -novid -high -map dota -noprewarm +map_enable_background_maps 0
> 
> ![Screen Shot 2023-04-21 at 2 34 10 pm](https://user-images.githubusercontent.com/131443012/233767603-8e84b6ed-281f-4fa8-b8bb-0314e870ff1c.png)
> 
> Edit: Ignore -console (i removed it before starting the game)

This enables opening dota, but it crashes again as soon as you join a game (or try to demo a hero).

**Comment by TooSpicyToWork** [comment]
> This enables opening Dota, but it crashes again as soon as you join a game (or try to demo a hero).
Huh, that's strange cause it worked for me. Did you try updating your Mac to the latest version beforehand? Because once i did that and started the game with just there were no issues. I only used these launch commands because the game crashed during/in the middle of gameplay because of low fps. In your case, however, maybe its something to do with the rendering API. I'm currently using Vulkan and it's working just fine.
